pkg/manifest/writer/manifest_writer_test.go

```diff
@@ -611,47 +611,6 @@ projects:
 - name: p1
   path: projects/p1
 environmentGroups:
-- name: group1
```

Comment:
note this part just happens to be the same in the following test-case so the diff looks weird. What's actually removed is the complete `"writes manifest 1.1 if account feature is active but account defined"` test case